test/model_simulation/simulate_ODEs.jl

```diff
@@ -103,26 +104,32 @@ let
 
     for (i, networks) in enumerate(identical_networks_1)
         for factor in [1e-2, 1e-1, 1e0, 1e1]
-            u0 = factor * rand(rng, length(get_unknowns(networks[1])))
-            p = factor * rand(rng, length(get_ps(networks[1])))
-            (i == 3) && (p = min.(round.(p) .+ 1, 10))                      #If parameter in exponent, want to avoid possibility of (-small u)^(decimal). Also avoid large exponents.
-            prob1 = ODEProblem(networks[1], u0, (0.0, 10000.0), p)
-            sol1 = solve(prob1, Rosenbrock23(), saveat = 1.0)
-            prob2 = ODEProblem(networks[2], u0, (0.0, 10000.0), p)
-            sol2 = solve(prob2, Rosenbrock23(), saveat = 1.0)
-            @test all(abs.(hcat((sol1.u .- sol2.u)...)) .< 1e-7)
+            u0_1 = rnd_u0(networks[1], rng; factor)
+            p_1 = rnd_ps(networks[1], rng; factor)
+            (i == 3) && (p_1 =rnd_ps_Int64(networks[1], rng))
+            u0_2 = last.(u0_1)
+            p_2 = last.(p_1)
```

Comment:
This depends on `ReactionSystem` not doing some weird reordering (which we do not want to promise in the future). However, here we loop through quite a few networks and just generate parameters. If we'd want to make order not matter here, we'd have to write out the vectors for each case manually, and handle them separately. I can see how to do this, but non-trivial amount of work. I'd rather do that at some later overhaul of the tests than right now (when we have tons of stuff to fix).